docker-compose-non-dev.yml

```diff
@@ -26,7 +26,7 @@ x-superset-volumes: &superset-volumes
 version: "3.7"
 services:
   redis:
-    image: redis:3.2
+    image: redis:6.2
```

Comment:
Let's change this to `:latest` so it's in sync with the [default docker-compose file](https://github.com/apache/superset/blob/master/docker-compose.yml#L32)

Reply:
actually, I'm about to make some changes to this file. I'll take care of it